### ISSUE TITLE
Can't see how to disable linter warnings on a line by line basis in axe Linter for VS-Code

### ISSUE THREAD

**Issue by Roger-Heathcote** [dequelabs/axe-core#3118]
I don't know if this is even possible. Sorry to ask here but your docs site is "undergoing maintenance".

https://axe-linter.deque.com/docs

**Comment by stephenmathieson** [comment]
Currently there is no way to disable a rule for a particular line, but to disable a rule for a file, do:


```js
/* axelinter:disable:RULE_NAME */
```

or in HTML/Markdown:

```html
<!-- axelinter:disable:RULE_NAME -->
```

For example:

```js
/* axelinter:disable:tabindex */

const Component = () => {
  return <a href="#" tabindex="100">Link</a>
}
```

**Comment by WilcoFiers** [comment]
I've opened a PR to add this to the VSC documentation. Thank you for reporting.
